demo/scss/demo.scss

```diff
@@ -68,6 +68,11 @@ REASON: #{$reason}';
     .bx--interior-left-nav.bx--interior-left-nav--collapsed {
       width: 3rem;
     }
+
+    &.link div {
+      display: grid;
```

Comment:
Does this work well for IE11...? Probably a flex box?

Reply:
I replaced the grid with flexbox